store/sqlstore/post_store.go

```diff
@@ -188,11 +200,16 @@ func (s *SqlPostStore) SaveMultiple(posts []*model.Post) ([]*model.Post, int, er
 	}
 
 	if len(unknownRepliesPosts) > 0 {
-		if err := s.populateReplyCount(unknownRepliesPosts); err != nil {
+		if err := s.populateReplyCount(transaction, unknownRepliesPosts); err != nil {
```

Comment:
Again, this was intentionally fire and forget, so shoulden't be part of the same transaction.